.github/scripts/google_oauth_cases.py

```
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+"""Select live-QA cases that can run without Google OAuth."""
+
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)